CheckpointPlugin.cpp

```diff
@@ -630,33 +715,32 @@ bool CheckpointPlugin::rewind(ServerWrapper sw) {
 	rewindState.buttonsDown = buttonsDown;
 
 	// Determine how much to rewind / advance time.
-	if (abs(ci.Steer) < .05f) { // Ignore slight input; keep current game state.
+	if (abs(activeInput) < .05f) { // Ignore slight input; keep current game state.
 		return true; // Ignoring input; apply state.
 	}
 	rewindState.deleting = false;
-	if (ci.Steer < -.95 && rewindState.holdingFor <= 0) {
+	if (activeInput < -.95 && rewindState.holdingFor <= 0) {
 		rewindState.holdingFor -= elapsed;
-	} else if (ci.Steer > .95 && rewindState.holdingFor >= 0) {
+	} else if (activeInput > .95 && rewindState.holdingFor >= 0) {
 		rewindState.holdingFor += elapsed;
 	} else {
 		rewindState.holdingFor = 0;
 	}
-	float factor = std::clamp(abs(rewindState.holdingFor) * 2, 1.0f, 10.0f);
-
-	// How much (in seconds) to move "current" (positive or negative)
-	float deltaElapsed = factor * elapsed * ci.Steer; // full left = 2-5 seconds/second
 
+	float factor = std::clamp(fabs(activeInput) * 2, 1.0f, 10.0f);
```

Comment:
`holdingFor` was used to make rewinding go faster over time.  This seems to have changed that behavior.